packages/preact-query/src/__tests__/usePrefetchInfiniteQuery.test.tsx

```
@@ -0,0 +1,204 @@
+import { fireEvent } from '@testing-library/preact'
+import { queryKey, sleep } from '@tanstack/query-test-utils'
+import {
+  QueryCache,
+  QueryClient,
+  usePrefetchInfiniteQuery,
+  useSuspenseInfiniteQuery,
+} from '..'
+import { renderWithClient } from './utils'
+import type { InfiniteData, UseSuspenseInfiniteQueryOptions } from '..'
+import type { Mock } from 'vitest'
+import { Suspense } from 'preact/compat'
```

ESLint

[error] `preact/compat` import should occur before import of `..`

(import/order)